src/log_setup.py

```diff
@@ -0,0 +1,47 @@
+import logging
+import os
+from datetime import datetime
+
+
+def setup_logger():
+    logger = logging.getLogger("dataFetcherLogger")
+    logger.setLevel(logging.DEBUG)
```

Comment:
This should have the ability to be toggled via an env variable or something. Generally we wouldn't want debug-level logging in production.

Reply:
Please create `.env.example` with an example of the env variable value. Also, update README to indicate that you can do this.